index.rst

```diff
@@ -179,68 +195,64 @@ Notes on "recommended" category
+If older releases are required, it may take some work, up to and including a separately-constructed RSP instance, to make them runnable.
+
+Historically, the "recommended" image during construction was usually a weekly; however, in operations, it will generally point to a release image.
 
 Tag Format
 ^^^^^^^^^^
 
 Release tags are now of the form ``r[major]_[minor]_[patch]``,
-e.g. ``r21_0_1``.  Prior to Release 18, they were not
-underscore-separated, e.g. ``r170``.  The first two digits are the major
-version, and the last one is the minor version.  In this form, the patch
-version is always 0.  Cycle and rest are permitted, so, for instance,
-all of ``r_21_0_1_c0019.001``, ``r_21_0_1_20210703``, and
-``r_21_0_1_c0019.001_20210703`` are allowed.
+e.g. ``r21_0_1``.
+Prior to Release 18, they were not underscore-separated, e.g. ``r170``.
+The first two digits are the major version, and the last one is the minor version.
+In this form, the patch version is always 0.
+RSP build version, cycle and rest are permitted, so, for instance, all of ``r_21_0_1_rsp9_c0019.001``, ``r_21_0_1_c0019.001``, ``r_21_0_1_20210703``, and ``r_21_0_1_rsp9_c0019.001_20210703`` are allowed.
 
 Display Name
 ^^^^^^^^^^^^
 
 The display name for a release is of the form ``Release
 r[major].[minor].patch``; thus ``r21_0_1`` has the display name ``Release
-r21.0.1``.  Additional components (cycle and extra) are permitted and
-will be appended in the following form: ``r21_0_1_c0020.002_20210703``
-becomes ``Release r21.0.1 (SAL Cycle 0020, Build 002) [20210703]``.
+r21.0.1``.
+Additional components (RSP version, cycle and extra) are permitted and will be appended in the following form: ``r21_0_1_rsp9_c0020.002_20210703``
+becomes ``Release r21.0.1 (RSP Build 9) (SAL Cycle 0020, Build 002) [20210703]``.
 
 Semantic Version
 ^^^^^^^^^^^^^^^^
 
-The semantic version of a release tag is, actually,
-``[major].[minor].[patch]``.  ``r21_0_1`` has version ``21.0.1``.
+The semantic version of a release tag is, actually, ``[major].[minor].[patch]``.  ``r21_0_1`` has version ``21.0.1``.
+
+Cycle and build version will be added as described above.
+Thus: ``r21_0_1_c0020.002_20210703`` would have the semantic version ``21.0.1+c0020.002.20210703``.
 
-Cycle and build version will be added as described above.  Thus:
-``r21_0_1_c0020.002_20210703`` would have the semantic version
-``21.0.1+c0020.002.20210703``.
+The RSP version is not included in the tag's semantic version (there is no room for it, and overloading the prerelease field yields undesirable sorting properties); it is, however, considered by the Nublado controller when constructing sort order.
+Specifically, it is considered after the primary version and before the cycle information when building a sort order.
+Any image lacking an RSP version sorts lower than an image with an RSP version.
+That is, "r21_0_1" will be sorted below "r21_0_1_rsp9".
```

Comment:
I think this is all material to move into an implementation section and appendix (and likewise below for other semver discussion), since most readers are going to care about the semantics of the version and not the implementation strategy.